### ISSUE TITLE
Cannot use "inlineSourceMap" or "inlineSources"

### ISSUE THREAD

**Issue by bryanforbes** [TypeStrong/grunt-ts#260]
When attempting to use inline source maps with inlining sources, I get an error:

```bash
error TS5048: Option 'sourceMap' cannot be specified with option 'inlineSourceMap'
```

Even if I explicitly set "sourceMap" to false in my options, it still gives this error. In the release, there is some code which I can't seem to find in the repo. In `ts.js` after the line `options.removeComments = !!options.removeComments;` there is this:

```js
            if (options.inlineSources) {
                options.inlineSourceMap = true;
            }
            if (options.inlineSourceMap) {
                options.sourceMap = true;
            }
```

This code is wrong. `--inlineSourceMap` cannot be passed with `--sourceMap` and `--inlineSources` can be used with either `--sourceMap` or `--inlineSourceMap` (even though the docs on the compiler options wiki page say otherwise. I'm not sure how that got into the release, but could we get another release without this code?

**Comment by nycdotnet** [comment]
Hello again - this code was implemented because in the TypeScript 1.5 beta it was required to pass the `sourceMap` flag to get the others working; from what you're saying, they reverted that decision in the version you have.  I would suggest using the `additionalFlags` feature of the beta release to pass `inlineSourceMap` for now.  When 1.5 is officially released, we'll tidy this all up.

**Comment by bryanforbes** [comment]
Ok. Thanks!

**Comment by bryanforbes** [comment]
In case you were wondering, I've worked around it in `dojo-core`'s [Gruntfile.js](https://github.com/dojo/core/blob/151676df1dfe39b3c2f97c51cdac541ccd1d887c/Gruntfile.js#L14-L34).

**Comment by nycdotnet** [comment]
Nice work.

**Comment by nycdotnet** [comment]
Hi Bryan,

This should be resolved with v4.2.0 published just now.   Please let me know if not.

Thanks for contributing!
